templates/stamp/utilities.sh

```diff
@@ -632,10 +632,23 @@ install-azure-cli()
         apt-get -y -qq update
 
         # Note: nodejs-legacy is required for Ubuntu14 and above.
-        log "Installing nodejs-legacy, npm, and azure cli"
-        apt-get install -y nodejs-legacy npm
-        exit_on_error "Failed to install nodejs-legacy and/or npm on ${HOSTNAME} !" $ERROR_NODEINSTALL_FAILED
+        short_release_number=`lsb_release -sr`
+        if [[ "$short_release_number" > 13 ]]; then
```

Comment:
definitely needed for azure cli 1.0 on Ubuntu 14+. i don't know if it's needed for cli 2.0 since that uses python. we can do some testing, but i know this conditional installation is helpful in many cases. this change fixes any problems I've seen with its installation